### PR TITLE
feat(#36): read users from json file instead of constant

### PR DESCRIPTION
**Pull request by sathia27**
Make the web app reload a json file for users list after a set interval of time #36

_Add details of your PR here.._

## Checklist
- [ ] Have you followed the [guidelines for contributing](https://github.com/javatarz/github-contribution-leaderboard/blob/master/contributing.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/javatarz/github-contribution-leaderboard/pulls) for the same feature you're adding?
- [ ] Have you run tests (`pipenv run pytest`) before sending this PR?
- [ ] Have you run linting (`./lint.sh`) before sending this PR?

### CODE REVIEW

**Comment by sathia27** [comment]
**How i tested?**
1. create new file `users.json`, and added few users in `users.json`
2. Hit request `http://127.0.0.1:5000/data.json`

**Output**
List of users with PR has been appeared

File: app.py [hunk 1]
```diff
@@ -26,3 +27,14 @@ def data():
 @app.route('/<path:path>')
 def send_js(path):
     return send_from_directory('web/src', path)
+
+def load_users():
+    users = []
+    users_file = "instance/users.json"
```
Comment (heaven00) [inline app.py]: Add this filename as part of `config.py`

File: app.py [hunk 2]
```diff
@@ -26,3 +27,14 @@ def data():
...
+    if path.isfile(users_file):
+        with open(users_file) as f:
+            try:
+                users = json.load(f)['users']
+            except ValueError as e:
+                return users
```
Comment (heaven00) [inline app.py]: replace the whole loop with `json.loads(open(users_file))`

File: app.py [hunk 3]
```diff
@@ -12,12 +14,11 @@
 @app.route('/data.json')
 def data():
     access_token = app.config['ACCESS_TOKEN']
-    users = app.config['USERS']
     start_date = valid_date(app.config['START_DATE'])
     end_date = valid_date(app.config['END_DATE'])
 
     contribs = Contributions(GithubStats(access_token=access_token))
-    stats = contribs.leaderboard(users, start_date, end_date)
+    stats = contribs.leaderboard(load_users(), start_date, end_date)
```
Comment (heaven00) [inline app.py]: stick to `stats = contribs.leaderboard(users, start_date, end_date)` 

and have users = load_users(app.config['user_file_path'])

**Comment by sathia27** [comment]
@heaven00 made few changes after ur PR. Let me know if anything needs to be done

**Comment by javatarz** [comment]
We've decided to use a HTTP endpoint to enter the list of users instead of a users.json file. The issue has been closed and a new one will be opened.

Thanks for your contribution.

**Pull request closed without merging**